yugabyte/src/yugabyte/workaround.clj

```diff
@@ -0,0 +1,44 @@
+(ns yugabyte.workaround
+  (:require [jepsen.checker :as checker]
+            [jepsen.checker.timeline :as timeline]
+            [hiccup.core :as h]
+            [jepsen.store :as store]
+            [knossos.history :as history]))
+
+(defn process-index
+  "Maps processes to columns"
+  [history]
+  (->> history
+       history/processes
+       history/sort-processes
+       (reduce (fn [m p] (assoc m p (count m)))
+               {})))
+
+(defn html
```

Comment:
It seems that the current implementation of `timeline/html` is not very different to this, why can't we use it?

Reply:
Trying to understand what's needs to be changed here https://github.com/jepsen-io/jepsen/issues/564
These  are the only tests (single-multi-key-acid) where checker is failed